### PR TITLE
BFD-458: Add test coverage for Synthea

### PR DESCRIPTION
**Pull request by karlmdavis**
### Change Details

Add test coverage for Synthea.

Adds support for generating data via a special branch/build of Synthea. For now, this data is just run through `RifLoaderIT` to verify it works.

This is a hilariously rough first draft, with a number of issues and `TODO`s noted in the changes. Notably, building Synthea is **slow**: takes 15-30 minutes on my laptop.

<https://jira.cms.gov/browse/BFD-458>

### Acceptance Validation

TODO

### Feedback Requested

TODO

### External References

- [BLUEBUTTON-xxx](https://jira.cms.gov/browse/BLUEBUTTON-xxx)

### Security Implications

TODO

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

Adds support for generating data via a special branch/build of Synthea. For now, this data is just run through RifLoaderIT to verify it works.

This is a hilariously rough first draft, with a number of issues and TODOs noted in the changes. Notably, building Synthea is slow: takes 15-30 minutes on my laptop.

### External References

- [BLUEBUTTON-xxx](https://jira.cms.gov/browse/BLUEBUTTON-xxx)

### Security Implications

- [ ] new software dependencies

- [ ] altered security controls

- [ ] new data stored or transmitted

- [ ] security checklist is completed for this change

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Comment by karlmdavis** [comment]
Couple notes:
1. This doesn't work right now, as Synthea's BFD export hasn't yet been updated to support some new columns that were recently added.
2. This replaces #347, which got a bit goofed during a merge.

**Review by cbrunefearless — APPROVED** [review]
lgtm

File: apps/bfd-model/bfd-model-rif-samples/dev/design-sample-data-sets.md [hunk 1]
```diff
@@ -449,3 +449,39 @@ It was created via the following process:
...
+  and generates a similarly wide range of FHIR data,
+  including `Patient`s, and `ExplanationOfBenfit` resources.
+That's great, but not super useful for BFD: BFD _produces_ FHIR;
+  it doesn't need or _consume_ FHIR data as input.
+
+In 2020, CMS engaged Synthea to add an output mode
+  that produced data in BFD's RIF input file formats.
+The initial engagement ended with these accomplishments:
+
+* Synthea added a `--exporter.bfd.export=true` option to produce RIF.
```
Comment (jzulim) [inline apps/bfd-model/bfd-model-rif-samples/dev/design-sample-data-sets.md:477]: Nice!

**Comment by christrotteradhoc** [comment]
closing this old PR since we merged BFD-458 under the new PR today

**Pull request closed without merging**